test/Parameters.js

```diff
@@ -33,6 +35,10 @@ describe('Parameters contract Tests', async () => {
   const gracePeriod = toBigNumber('8');
   const minimumStake = tokenAmount('100');
   const aggregationRange = toBigNumber('3');
+  const withdrawReleasePeriod = toBigNumber('5');
+  const resetLockPenalty = toBigNumber('1');
+  const maxAge = toBigNumber('1000000');
+  const maxAssetsPerStaker = toBigNumber('2');
```

Comment:
@GauravJain9 Please make changes according to latest merge which reverts assign assets randomly.

Reply:
Yeah I did now @SkandaBhat